packages/client/src/hooks/useTreasury.js

```diff
@@ -68,11 +70,9 @@ const doProposeTransfer = async (
   coinType
 ) => {
   const uFixAmount = String(parseFloat(amount).toFixed(8));
-  const tokenAddress =
-    process.env.REACT_APP_FLOW_ENV === "emulator"
-      ? "ee82856bf20e2aa6"
-      : "9a0766d93b6608b7";
-  const recepientVault = `Capability<&AnyResource{A.${tokenAddress}.FungibleToken.Receiver}>`;
+  const contractAddress = await config().get(`0x${COIN_TYPE_TO_META[coinType].contract}`);
```

Comment:
Can we consider add a safe check for `COIN_TYPE_TO_META[coinType]`? I think it's unlikely to happen in current workflow but it's a good practice to error out explicitly in case input coinType is not one of the supported coin.

Reply:
It can happen if the user tampers with the html, and those errors are their fault 😄 
Is it enough to add an error in the console or we would need to present an error on the UI?

Reply:
i think a console log is good enough for now.

Reply:
Added error handling as part of proposeAddVault and proposeAddCollection -> error will be logged to the console error in the browser